index.js

```diff
@@ -0,0 +1,69 @@
+'use strict';
+
+const differenceWith = require('lodash/differenceWith');
+const isMatch = require('lodash/isMatch');
+
+function isAction(value) {
+    return typeof value === 'object' && typeof value.type === 'string';
+}
+
+const wait = (store, actions, timeout) => {
+    if (!timeout) {
+        timeout = 50;
+    }
+
+    if (typeof actions !== 'function') {
+        if (!Array.isArray(actions)) {
+            actions = [actions];
+        }
+        actions = actions.map((value) => {
```

Comment:
I would convert this to one line: `typeof value === 'string' ? { type: value } : value`